### DISCUSSION TITLE
Affiliate Links Policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#42] [Rejected]
### Type name

Affiliate Links Policy

### Alternative type names

Editorial Policy

### Writer

service provider

### Audience

end user

### Object

use, disclosure, and earning from affiliate links

### Examples

| Example | Service Name | URL | Title | Jurisdiction |
|:----|:----|:----|:------|:-------------|
| 1 | Ars Technica | [URL 🔗](https://arstechnica.com/affiliate-link-policy/) | Affiliate Link Policy | - |
| 2 | PCWorld | [URL 🔗](https://www.pcworld.com/about/affiliate-link-policy) | Editorial Independence | - |
| 3 | Well Life Therapy | [URL 🔗](https://welllifetherapyllc.com/affiliate-link-policy) | Affiliate Link Policy | - |
| 4 | Time | [URL 🔗](https://time.com/4605406/time-affiliate-link-policy/) | Affiliate Link Policy | - |
| 5 | Coffee or Die’s | [URL 🔗](https://coffeeordie.com/affiliate-policy) | Affiliate Links Policy | - |
| 6 | Wired | [URL 🔗](https://www.wired.com/2015/11/affiliate-link-policy/) | Editorial Policy on Affiliate Links | - |

**Comment by MattiSG** [comment]
This looks good! I'm not convinced by the alternate name “Editorial Policy” as it is too broad.

I wonder however if “affiliate links” are not just a specific technical implementation of product placement or advertisement policy, both of which we are lacking. For example, it could include “Branded Content Policy” ([Instagram](https://help.instagram.com/1695974997209192), [TikTok](https://www.tiktok.com/legal/page/global/bc-policy/en), [Twitch](https://help.twitch.tv/s/article/branded-content-policy?language=en_US)).

The generic type could be “Affiliate Content Policy” or “Branded Content Policy”. Though I agree that an affiliate link and a dedicated content are different and the usual policies are opposite, I feel they are close enough to justify unifying them. Are there cases where both policies would co-exist in the same service?

**Reply by MattiSG** [reply]
Checking on the examples, only “Coffee or Die” has both an [Editorial Policy](https://coffeeordie.com/editorial-standards) and an [Affiliate Links Policy](https://coffeeordie.com/affiliate-policy). All others have only an Affiliate Links Policy, which could thus be described as an Editorial Policy. For Coffee or Die and other similar services with both policies, one can just make a multi-document entry for Editorial Policy, thus including the Affiliate Links Policy.

I thus feel that #35 would be a more appropriate, broader terms type. I vote for including “Affiliate Links Policy” as an alternative name to #35 and closing #42.

**Reply by MattiSG** [reply]
Going through the examples in #35, I get reinforced in the idea that Affiliate Links Policy is a specific subset of Editorial Policy that should not have its own type:

Editorial Policy | Title | Service | Has Affiliate Links Policy
--------------|----|-----|-----
[bbc.com/editorialguidelines/guidelines](https://www.bbc.com/editorialguidelines/guidelines) | Editorial Guidelines | BBC | Yes, included in [section 16.D](https://www.bbc.com/editorialguidelines/commercial-services/)
[misinforeview.hks.harvard.edu/editorial-policies](https://misinforeview.hks.harvard.edu/editorial-policies/) | Editorial Policies | HKS Misinformation Review | No
[bleepingcomputer.com/ethics-statement](https://www.bleepingcomputer.com/ethics-statement/) | ETHICS STATEMENT | BleepingComputer | Yes, included
[lawandcrime.com/ethics-diversity-policy](https://lawandcrime.com/ethics-diversity-policy/) | Corrections, Ethics & Diversity Policy | Law&Crime | No
[themarkup.org/ethics](https://themarkup.org/ethics) | Ethics Policy | The Markup | No
[bioresources.cnr.ncsu.edu/about-the-journal/editorial-policies](https://bioresources.cnr.ncsu.edu/about-the-journal/editorial-policies/) | Editorial Policies | BioResources | No
[washingtonpost.com/policies-and-standards](https://www.washingtonpost.com/policies-and-standards/) | Policies and Standards | Washington Post | No

**Reply by clementbiron** [reply]
ok I agree with that

**Comment by clementbiron** [comment]
This suggestion has been rejected but please feel free to comment if you would like the community to re-evaluate this decision.

**Reply by MattiSG** [reply]
To be more specific: if you would like to use this terms type, the current recommended practice is to use Editorial Policy (#35) 🙂
